cluster/gce/util.sh

```diff
@@ -1344,11 +1349,7 @@ function ssh-to-node {
 
 # Restart the kube-proxy on a node ($1)
 function restart-kube-proxy {
-  if [[ "${OS_DISTRIBUTION}" == "trusty" ]]; then
-    ssh-to-node "$1" "sudo initctl restart kube-proxy"
-  else
-    ssh-to-node "$1" "sudo /etc/init.d/kube-proxy restart"
-  fi
+  ssh-to-node "$1" "sudo /etc/init.d/kube-proxy restart"
```

Comment:
I wouldn't worry about changing this; see https://github.com/kubernetes/kubernetes/pull/23180